### ISSUE TITLE
TypeError: The "original" argument must be of type function. Received undefined

### ISSUE THREAD

**Issue by kodelio** [sequelize/umzug#651]
Hello,

I'm trying to run migrations, and while locally I have no error, I get this error on the deployment server :

```
TypeError: The "original" argument must be of type function. Received undefined
    at promisify (node:internal/util:410:3)
    at Object.<anonymous> (/usr/src/app/node_modules/umzug/src/umzug.ts:26:28)
    at Module._compile (node:internal/modules/cjs/loader:1376:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1435:10)
    at Object.require.extensions.<computed> [as .js] (/home/node/.npm/_npx/1bf7c3c15bf47d04/node_modules/ts-node/src/index.ts:1608:43)
    at Module.load (node:internal/modules/cjs/loader:1207:32)
    at Function.Module._load (node:internal/modules/cjs/loader:1023:12)
    at Module.require (node:internal/modules/cjs/loader:1235:19)
    at require (node:internal/modules/helpers:176:18) {
  code: 'ERR_INVALID_ARG_TYPE'
}
```

Here's my code :

migrate.ts
```
import { umzug } from './umzug-wrapper'
// eslint-disable-next-line import/newline-after-import
;(async () => {
  await umzug.up()
})()
```

umzug-wrapper.ts
```
import { SequelizeStorage, Umzug } from 'umzug'

import { sequelizeInstance } from '../../../infrastructure/dependencies/dependencies-repositories'

export const umzug = new Umzug({
  migrations: { glob: `${__dirname}/../migrations/*.[jt]s` },
  create: {
    folder: `${__dirname}/../migrations/`,
  },
  context: sequelizeInstance.sequelize.getQueryInterface(),
  storage: new SequelizeStorage({ sequelize: sequelizeInstance.sequelize }),
  logger: console,
})
```

I cannot find what is throwing this error

**Comment by kodelio** [comment]
@WikiRik we have glob 10.3.10 installed (because required by other packages)

**Comment by mmkal** [comment]
@kodelio could you share your setup - which package manager are you using? npm, yarn, pnpm, bun? As well as which version + are you using any overrides, `.pnpmfile.cjs` etc.?

And/or if you can share your whole package.json that'd be even better. It doesn't sound like a bug in umzug, but maybe we can help.

**Comment by kodelio** [comment]
Issue has been fixed by adding glob 10.3.10 in the package.json, thank you
